samples/sample_multi_transcode/include/pipeline_transcode.h

```diff
@@ -664,9 +662,18 @@ namespace TranscodingSample
         virtual mfxStatus PreEncPreInit(sInputParams *pParams);
 
         mfxVideoParam GetDecodeParam();
-        mfxExtOpaqueSurfaceAlloc GetDecOpaqueAlloc() const { return m_pmfxVPP.get() ? m_VppOpaqueAlloc: m_DecOpaqueAlloc; };
 
-        mfxExtMVCSeqDesc GetDecMVCSeqDesc() const {return m_MVCSeqDesc;}
+        mfxExtOpaqueSurfaceAlloc GetDecOpaqueAlloc()
+        {
+            mfxExtOpaqueSurfaceAlloc* opaq = m_pmfxVPP.get() ? m_mfxVppParams : m_mfxDecParams;
```

Comment:
no need to call ```.get()``` to check pointer. all smart pointers have ```bool operator ()``` which makes their behavior really similar to regular pointers

Reply:
ok